odoo/cli/shell.py

```
@@ -78,6 +78,11 @@ def console(self, local_vars):
             else:
                 shells_to_try = self.supported_shells
 
+            if pythonstartup := config.options.get('shell_file') or os.environ.get('PYTHONSTARTUP'):
+                if not os.path.exists(pythonstartup):
```

Why? Let it crash if the file does not exist.